jb-lib/lib/channel_handler/channel_handler.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Generator, Optional
+from pydantic import BaseModel
+from lib.models import JBChannel, JBUser
+from lib.data_models import Message, BotInput
+
+
+class User(BaseModel):
+    user_identifier: str
+    user_name: str
+    user_email: Optional[str] = None
+    user_phone: Optional[str] = None
+
+
+class ChannelData(BaseModel):
+    bot_identifier: str
+    user: User
+    message_data: Dict
+
+
+class ChannelHandler(ABC):
```

Comment:
Write docstrings for each of these functions so that we are clear what their scope is

Reply:
Added